native_client/deepspeech.cc

```diff
@@ -1,4 +1,7 @@
 #include <algorithm>
+#ifdef _MSC_VER
+  #define _USE_MATH_DEFINES
+#endif
 #include <cmath>
```

Comment:
Why cmath here and math in the [c_speech_features.cpp](https://github.com/carlfm01/DeepSpeech/blob/68905f72296428e2935f9c3c814bb82a575609e8/native_client/c_speech_features/c_speech_features.cpp#L5) ?